### DISCUSSION TITLE
nextcloud-aio-talk-recording not working

### DISCUSSION THREAD

**Discussion by devinnovqube** [nextcloud/all-in-one#5406] [Questions] [answered]
Hello,

AIO working fine except **nextcloud-aio-talk-recording**.
Recording don't start when in a call and i have cURL error message in log.
Issue came since first day. No other issue in any other addons (whiteboard for last or talk work fine)

**Version:** 
AIO : v9.6.0 latest

**Error message in nextcloud log :**

"method":"POST","url":"/ocs/v2.php/apps/spreed/api/v1/recording/xxx","message":"cURL error 7: **Failed to connect to nextcloud-aio-talk-recording port 1234 after 16 ms**: Could not connect to server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36","version":"29.0.7.1","exception"

**Nextcloud Talk recording configuration panel :**
Url : http://nextcloud-aio-talk-recording:1234
Key : same as in /conf/recording.conf of the container.
ssl : verification off

**Log from nextcloud-aio-talk-recording container :**
2024-10-10T23:14:02.038408255Z  * Serving Flask app 'nextcloud.talk.recording.Server'
2024-10-10T23:14:02.038448683Z  * Debug mode: off

**TCP communication from nextcloud/aio-nextcloud:latest work fine :**
```
# wget http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx
Connecting to nextcloud-aio-talk-recording:1234 (172.18.0.4:1234)
wget: server returned error: HTTP/1.1 405 METHOD NOT ALLOWED
```

nextcloud-aio-talk-recording and nextcloud/aio-nextcloud  are pinging each other.

**Configuration :**
- Proxmox
- LXC container
- Enough Ram and CPU
- NVME Ceph

Thanks for reading.

**Comment by devinnovqube** [comment]
**additional info :**

Wget :
```
# wget http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx
Connecting to nextcloud-aio-talk-recording:1234 (172.18.0.4:1234)
wget: server returned error: HTTP/1.1 405 METHOD NOT ALLOWED
```

curl :
```
# curl http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx
curl: (7) Failed to connect to nextcloud-aio-talk-recording port 1234 after 19 ms: Could not connect to server
```

Curl work in shell after upgrade curl (curl 8.9.1  to curl 8.10.1) : 
```
# apk upgrade --no-cache -a
```

```
# curl http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx
<!doctype html>
<html lang=en>
<title>405 Method Not Allowed</title>
<h1>Method Not Allowed</h1>
<p>The method is not allowed for the requested URL.</p>
```

**But still same error in log :**
cURL error 7: Failed to connect to nextcloud-aio-talk-recording port 1234 after 20 ms: Could not connect to server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx","exception":{},"CustomMessage":"cURL error 7: Failed to connect to nextcloud-aio-talk-recording port 1234 after 20 ms: Could not connect to server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://nextcloud-aio-talk-recording:1234/api/v1/room/xxx"}}

**Reply by devinnovqube** [reply, marked as answer]
new docker image with updated alpine seems fix the issue. Thanks

**Comment by szaimen** [comment]
Hi, can you check if upgrading to v9.7.0 Beta resolves this? See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-switch-the-channel
